core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -263,12 +263,12 @@ private[query] class EventsByTagPublisher(
     case ReplayDone(retrievedCount, deliveredCount, seqN, highest, mightBeMore) =>
       if (log.isDebugEnabled)
         log.debug(
-          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}]",
+          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}], mightBeMore [$mightBeMore]",
           tag, currTimeBucket, deliveredCount, retrievedCount
         )
       seqNumbers = seqN
       currOffset = highest
-      replayCountZero = retrievedCount == 0
+      replayCountZero = backtrackingMode == NoBacktracking && retrievedCount == 0
```

Comment:
We want to reach the end condition (`replayCountZero == true`) by a normal query not via a LookingForDelayed query.
However, we should disable the LookingForDelayed altogether for currentEventsByTag.